HISTORY.md

```diff
@@ -1,8 +1,9 @@
 # cloudpathlib Changelog
 
-## UNRELEASED
+## v0.20.0 (2024-10-18)
 
 - Added support for custom schemes in CloudPath and Client subclases. (Issue [#466](https://github.com/drivendataorg/cloudpathlib/issues/466), PR [#467](https://github.com/drivendataorg/cloudpathlib/pull/467))
+- Add check for HNS that works in more scenarios and with different kinds of credentials. (Issue [#470](https://github.com/drivendataorg/cloudpathlib/issues/470), Issue [#476](https://github.com/drivendataorg/cloudpathlib/issues/476), PR [#478](https://github.com/drivendataorg/cloudpathlib/pull/478))
```

Comment:
I think this is the wrong approach to writing changelog entries. I think it should be something like: 

> Fixed errors that could occur when instantiating `AzureBlobClient` under certain authentication approaches. 

What you have right now describes the implementation detail. The thing that matters for users is the runtime errors. I'm imagining a user who is using v0.19.0 and gets an error, and can see in the changelog for v0.20.0 that it might be fixed. (It would be better if we identify what those errors are, but maybe this would be annoying to actually list out.)